src/components/players/player/player.tsx

```diff
@@ -28,13 +29,41 @@ export const Player: FC<Props> = ({
   hasGuessed,
   isBot,
   onSelectPlayer,
+  onHoverPlayer,
   stage = "chat",
 }) => {
   const { name, color } = character;
   const textColor = `${color}.main`;
 
   const isBotSelected = stage === "voting" && isSelected;
 
+  const containerRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    const onMouseEnter = () => {
+      if (onHoverPlayer) {
```

Comment:
Why do you need to check `onHOverPlayer`? Can this be undefined?

Reply:
Yes it can be undefined... this `player` component is used by quite a few other components which do not use a on hover state.